package.json

```diff
@@ -91,6 +91,11 @@
       }
     ]
   },
+  "jest": {
+    "coveragePathIgnorePatterns": [
+      "test"
```

Comment:
I'd ignore fixtures instead

Reply:
But fixtures aren't included by default AFAICT.

Reply:
![image](https://user-images.githubusercontent.com/5635476/133087027-81c0e232-8975-4e87-a751-e8b3a98e6687.png)

Reply:
Yeah, which is what this PR ignores.

Reply:
All files here are in fixtures folders. So looks like they are not igored.

Reply:
They are 100% ignored... https://github.com/svg/svgo/pull/1567/checks?check_run_id=3586856059

Reply:
> But fixtures aren't included by default AFAICT.

"By default" they are included. And I'm asking to ignore fixtures instead of test

Reply:
Personally, I see no point in including the test folder in coverage. And fixtures right now are in the test folder which is ignored.

Reply:
Tests are ignored by default. We ignoring here not tests but fixtures specifically. They are not important to cover. Test utils are better be checked for example as a way to find dead code.

Reply:
Doesn't seem to be possible with `coverageIgnorePattern` or at least I can't make it work to ignore only the test files instead of the folder.

Reply:
"fixtures", not tests